kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorCreateView.java

```diff
@@ -35,7 +35,7 @@ public ConnectorsView setConnectorConfig(String connectName, String configJson)
     }
 
     @Step("Verify that page 'Create Connector' opened")
```

Comment:
need to fix step description: 'Wait until page opened'

Reply:
@anezboretskiy better just delete description from annotation

Reply:
@VladSenyuta description deleted